magnum_cluster_api/resources.py

```diff
@@ -750,6 +750,43 @@ def get_object(self) -> objects.ClusterClass:
                                 },
                             },
                         },
+                        {
+                            "name": "apiServerOIDC",
+                            "required": True,
+                            "schema": {
+                                "openAPIV3Schema": {
+                                    "type": "object",
+                                    "required": [
+                                        "oidcIssuerUrl",
+                                        "oidcClientId",
+                                        "oidcUsernameClaim",
+                                        "oidcUsernamePrefix",
+                                        "oidcGroupsClaim",
+                                        "oidcGroupsPrefix",
```

Comment:
these all seem to be redundant that they are prefixed with `oidc` since the top level object is `apiServerOIDC`.

Last small thing, could you rename these to be not prefixed:

```suggestion
                                        "issuerUrl",
                                        "clientId",
                                        "usernameClaim",
                                        "usernamePrefix",
                                        "groupsClaim",
                                        "groupsPrefix",
```